Sources/MapboxDirections/Directions.swift

```diff
@@ -536,6 +536,8 @@ open class Directions: NSObject {
      - returns: The URL to send the request to.
      */
     open func url(forCalculating options: DirectionsOptions, httpMethod: String) -> URL {
+        if options.waypoints.count < 2 { assertionFailure("waypoints array requires at least 2 waypoints") }
```

Comment:
What URL can we return here? We can't make this method returning optional URL, but there is no valid URL for the case when `options.waypoints.count < 2`.

Reply:
Unfortunately, making the return type optional would be a backwards-incompatible change. Should we shift the check to where Directions calls this method for now?